interface/frontend/src/components/detect-and-select/CandidateList.vue

```diff
@@ -11,7 +11,13 @@
                       'candidate-marked': candidate.review_result === REVIEW_RESULT.MARKED}">
                     <span class="candidate-title">
                       Candidate {{ index + 1 }}
-                    </span> (p={{ candidate.probability_concerning.toFixed(3) }})
+                    </span>
+                    <span v-if="candidate.probability_concerning">
```

Comment:
When would it be null out of interest?

Reply:
When `added_by_hand` - the `probability_concerning` always comes from the models. You _could_ set it to 1.0 if hand labeled but (a) perfect certainty isn't necessarily a good representation of what the clinician believes, and (b) we already handle human-labeled P(concerning) in the next screen.